vdev/src/commands/release/homebrew.rs

```diff
@@ -0,0 +1,80 @@
+use anyhow::{Result};
+use std::env;
+use tempfile::TempDir;
+use crate::git;
+use hex;
+use sha2::Digest;
+use reqwest::blocking::get;
```

Comment:
Nit: I prefer not importing functions into the namespace to avoid confusing them for local functions.